src/wallet/wallet2.cpp

```diff
@@ -6317,6 +6323,11 @@ bool wallet2::parse_unsigned_tx_from_str(const std::string &unsigned_tx_st, unsi
       LOG_PRINT_L0("Failed to decrypt unsigned tx: " << e.what());
       return false;
     }
+    catch(...)
+    {
+    LOG_PRINT_L0("Failed to parse data from unsigned tx");
```

Comment:
This message should probably be "Failed to decrypt unsigned tx", shouldn't it?

Reply:
Yeah makes sense to have it the same as your new message.